jupyterhub/templates/hub/deployment.yaml

```diff
@@ -110,6 +110,8 @@ spec:
           env:
             - name: PYTHONUNBUFFERED
               value: "1"
+            - name: HELM_RELEASE_NAME
+              value: {{ .Release.Name | quote }}
```

Comment:
Perhaps we should pass the [built in objects](https://docs.helm.sh/chart_template_guide/#built-in-objects) as well in the configmap?

Reply:
We could pass:
- Capabilities, Chart and Release
But not:
- Files and Template

Reply:
I initially tried this, but discovered that we actually *shouldn't* do that, at least for release and chart. Release is different on every deploy (it includes timestamps) and a change to this configmap triggers a Hub relaunch, which we don't want. So instead, I included the only value that is actually used via an environment variable. Plus, I can't think of any other subsets of this that we should be using in the deployment, and if it turns out there's another item we need, we can add it in the future.

We could include release & chart in a *different* configmap that doesn't trigger relaunch, if we find we need more than one key out of them.